shaman/controllers/root.py

```diff
@@ -1,8 +1,20 @@
 from pecan import expose
 
+from shaman.controllers import repos
+
 description = "shaman is the source of truth for the state of repositories on chacra nodes."
 
 
+class APIController(object):
+    @expose('json')
+    def index(self):
+        return dict(
+            repos="",
```

Comment:
can we return something a bit better here? maybe not repos because that is what the Repos controller is for. But some information about the current state might be cool. Anything/everything that may sound interesting:

* last repo udpated/created
* total repository count

etc..